CMakeLists.txt

```diff
@@ -330,7 +330,10 @@ function(feature_probe PROBE_NAME)
         CMAKE_FLAGS ${ADDITIONAL_FLAGS}
         COMPILE_DEFINITIONS -c ${GLOBAL_FLAGS} ${PROBE_FLAGS}
         ${ARGN}
+        OUTPUT_VARIABLE TRY_COMPILE_OUTPUT
     )
+    # Uncomment the line below to get the output of the try_compile command
+    #message(VERBOSE "Output of try_compile: ${TRY_COMPILE_OUTPUT}")
```

Comment:
Normally I'm against commenting out code, but yes please this is always so annoying to debug!

Although since you set it as VERBOSE, can we use CMAKE_MESSAGE_LOG_LEVEL to enable / disable it instead?